nsxt/resource_nsxt_policy_host_switch_profile.go

```diff
@@ -0,0 +1,592 @@
+			"revision":     getRevisionSchema(),
+			"tag":          getTagsSchema(),
+			"resource_type": {
+				Type:         schema.TypeString,
+				Description:  "Host switch profile type",
+				Required:     true,
+				ValidateFunc: validation.StringInSlice(policyHostSwitchProfileResourceType, false),
+			},
+			"uplink_host_switch_profile": getUplinkHostSwitchProfileSchema(),
+		},
+	}
+}
+
+func getUplinkHostSwitchProfileSchema() *schema.Schema {
+	return &schema.Schema{
+		Type:        schema.TypeSet,
+		Description: "Uplink host switch profile",
+		Optional:    true,
+		MaxItems:    1,
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				"lags": {
+					Type:        schema.TypeSet,
+					Description: "List of LACP group",
+					Optional:    true,
+					MaxItems:    64,
+					Elem: &schema.Resource{
+						Schema: map[string]*schema.Schema{
+							"id": {
+								Type:        schema.TypeString,
+								Description: "unique id",
+								Required:    false,
+								Optional:    false,
+								Computed:    true,
+							},
+							"load_balance_algorithm": {
+								Type:         schema.TypeString,
+								Description:  "LACP load balance Algorithm",
+								Required:     true,
+								ValidateFunc: validation.StringInSlice(lacpLoadBalanceAlgorithms, false),
+							},
+							"mode": {
+								Type:         schema.TypeString,
+								Description:  "LACP group mode",
+								Required:     true,
+								ValidateFunc: validation.StringInSlice(lacpGroupModes, false),
+							},
+							"name": getDataSourceRequiredStringSchema("Lag name"),
+							"number_of_uplinks": {
+								Type:         schema.TypeInt,
+								Description:  "Number of uplinks",
+								Required:     true,
+								ValidateFunc: validation.IntBetween(2, 32),
+							},
+							"timeout_type": {
+								Type:         schema.TypeString,
+								Description:  "LACP timeout type",
+								Optional:     true,
+								Default:      model.Lag_TIMEOUT_TYPE_SLOW,
+								ValidateFunc: validation.StringInSlice(lacpTimeoutTypes, false),
+							},
+							"uplinks": {
+								Type:        schema.TypeSet,
+								Description: "uplink names",
+								Required:    false,
+								Optional:    false,
+								Computed:    true,
+								Elem: &schema.Resource{
+									Schema: map[string]*schema.Schema{
+										"uplink_name": getDataSourceRequiredStringSchema("Name of this uplink"),
+										"uplink_type": {
+											Type:         schema.TypeString,
+											Description:  "Type of the uplink",
+											Required:     true,
+											ValidateFunc: validation.StringInSlice(uplinkTypes, false),
+										},
+									},
+								},
+							},
+						},
+					},
+				},
+				"teaming": {
+					Type:        schema.TypeSet,
+					Description: "Default TeamingPolicy associated with this UplinkProfile",
+					Required:    true,
+					MaxItems:    1,
+					Elem: &schema.Resource{
+						Schema: map[string]*schema.Schema{
+							"active_list": {
```

Comment:
`active_list` => `active`
in terraform config each `active` would appear in separate sub-clause, so `list` wouldn't make sense in the name

Reply:
Make sense. Done.